aws/templates/solution/solution_mobilenotifier.ftl

```diff
@@ -0,0 +1,206 @@
+[#-- Mobile Notifier --]
+
+[#if componentType == MOBILENOTIFIER_COMPONENT_TYPE  ]
+
+    [#list requiredOccurrences(
+            getOccurrences(tier, component),
+            deploymentUnit) as occurrence]
+
+        [@cfDebug listMode occurrence false /]
+
+        [#assign core = occurrence.Core]
+        [#assign solution = occurrence.Configuration.Solution]
+        [#assign resources = occurrence.State.Resources]
+
+        [#assign successSampleRate = solution.SuccessSampleRate ]
+        [#assign encryptionScheme = solution.Credentials.EncryptionScheme?ensure_ends_with(":")]
+
+        [#assign deployedPlatformAppArns = []]
+
+        [#assign roleId = resources["role"].Id]
+
+        [#if deploymentSubsetRequired("iam", true) && isPartOfCurrentDeploymentUnit(roleId)]
```

Comment:
Do we need to predefine the log groups like we did with lambda to be able to configure the monitors?

Is there any ability to control the log group name?

Reply:
Ahh yeah good point, no you can't change it. Problem I had was that I hadn't set a notification to find out what the log path was and can't find it documented. 

If we merge this PR in then I can add one after its been tested completely.